gatsby-config.ts

```diff
@@ -194,7 +194,7 @@ const config: GatsbyConfig = {
     {
       resolve: "@chakra-ui/gatsby-plugin",
       options: {
-        resetCSS: false,
+        resetCSS: true,
```

Comment:
We have enabled the reset from Chakra. It does seem to be working fine so far. I haven't seen any conflict yet.